config/upstart.example/tfb.conf

```diff
@@ -0,0 +1,18 @@
+# /etc/init/tfb.conf
+env TFB_REPOPARENT="/private"
+env TFB_REPONAME="FrameworkBenchmarks"
+env TFB_REPOURI="git@github.com:ashawnbandy-te-tfb/FrameworkBenchmarks.git"
```

Comment:
HOWEVER, my actually valuable comment on this chain is that TFB_REPOURI should probably be the https URL rather than the ssh URL. In my local testing, I was getting public key permission errors trying to clone down from your fork.